app/assets/stylesheets/sbstyles.scss

```diff
@@ -0,0 +1,26 @@
+body {
+  background-image: image-url("background1.jpg");
+  font-family: "Karma", serif;
+}
+
+h1 {
+  color: #df7401;
+  font-weight: bold;
+}
+
+h2 {
+  color: #faac58;
+  font-weight: bold;
+}
+
+.top-bar {
+  margin-bottom: 10px;
+}
+
+.button {
+  margin-right: 10px;
+}
+
+.button a {
+  color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.